sormas-ui/src/main/java/de/symeda/sormas/ui/configuration/infrastructure/AreasView.java

```diff
@@ -177,7 +180,11 @@ private HorizontalLayout createFilterBar() {
 				filterRelevanceStatus.setId("relevanceStatus");
 				filterRelevanceStatus.setWidth(220, Unit.PERCENTAGE);
 				filterRelevanceStatus.setEmptySelectionAllowed(false);
-				filterRelevanceStatus.setItems((EntityRelevanceStatus[]) EntityRelevanceStatus.getAllExceptDeleted());
+				List<EntityRelevanceStatus> items = new ArrayList<>();
+				for (Object o : EntityRelevanceStatus.getAllExceptDeleted()) {
+					items.add((EntityRelevanceStatus) o);
+				}
+				filterRelevanceStatus.setItems(items);
```

Comment:
why don't you just use `filterRelevanceStatus.addItems` like on the other views?